core/src/com/unciv/logic/automation/ConstructionAutomation.kt

```diff
@@ -187,6 +186,11 @@ class ConstructionAutomation(val cityConstructions: CityConstructions){
         if (preferredVictoryType == VictoryType.Cultural
                 && wonder.name in listOf("Sistine Chapel", "Eiffel Tower", "Cristo Redentor", "Neuschwanstein", "Sydney Opera House"))
             return 3f
+        // Only start building if we are the city that would complete it the soonest
+        if (wonder.name == "Utopia Project" && cityInfo == civInfo.cities.minByOrNull { 
```

Comment:
I know that there are other buildings referenced by name, but that's an antipattern - buildings should be referenced by unique, to enable modding

Reply:
I'll change this